src/api.v2/helpers/s3/bucketNames.js

```diff
@@ -0,0 +1,7 @@
+const config = require('../../../config');
+
+const bucketNames = {
+  SAMPLE_FILES: `biomage-originals-${config.clusterEnv}`,
+};
+
```

Comment:
why in helpers folder instead of config folder? Bucket names are a type of config

Reply:
`tableNames.js` inside `model` could go to `config` too.

Reply:
Actually I'm not sure where to put these, wouldn't they fit more into constants?

Reply:
Are you talking about moving it to this constants file?
<img width="760" alt="Screenshot 2022-04-26 at 16 44 29" src="https://user-images.githubusercontent.com/65414651/165313886-348e7d9f-e867-474c-ae5c-bf84139be018.png">

Reply:
Because this doesn't look fitting for bucket names or aws resources names

Reply:
Unrelated, but also I'm thinking to rename `helpers` to `services`, I think it fits better and is based on this kind of design

Reply:
no, more like a folder for constants